domains/certificates/Query.API/API/Program.cs

```diff
@@ -47,6 +50,13 @@
     });
 });
 
+builder.Services.AddMarten(options =>
```

Comment:
This only adds Marten to the application, but it is not used by the application in this PR. Do you know/have you tested that the application in the preview-environment can actually connect to the database?